src/components/Sortable.vue

```diff
@@ -73,6 +73,8 @@ const getKey = computed(() => {
   return props.itemKey;
 });
 
+defineExpose({ containerRef, sortable: <unknown>sortable });
```

Comment:
No, it throws type error, because the real type is _Ref<UnwrapRef<Sortable>>_, but this type throws the next error: _error TS4082: Default export of the module has or is using private name 'Sortable'._

Reply:
I changed type to `<Ref<Sortable | null>>`